main.yml

```diff
@@ -935,6 +935,20 @@ storybook:
       - /docs/storybook
   source_repo: https://github.com/RedHatInsights/insights-frontend-storybook
 
+subscription-central:
```

Comment:
This should be `manifests` so the subs navigates user to this application.

Reply:
Does the app have API? If so it's good to include the api part as well so the docs are autogenerated from the openapi.json spec.

Reply:
@karelhala would you rather use manifests here? I added their identifier in their UI as `subscription-central`

Reply:
Subscription-central may be better, since long-term (from my understanding) this intends to have more than just manifests.  But let me know if that creates an issue with routing.

The app itself is just a front-end; it gets data from RHSM-API - https://api.access.redhat.com/management/v1/allocations; does that need to be registered in this?

Reply:
You can keep the `subscription-central` as app name, however you'll have to change in the subs section https://github.com/RedHatInsights/cloud-services-config/blob/c4661b14d4c9ed88d3b44ad937b130bd8092ad82/main.yml#L985-L986

To be
```YAML
      - id: subscription-central
```

Reply:
I think the line that you highlighted is actually part of the Subscription Watch app's config, not this new Subscription Central app.  So I don't think we should change that.  Let me know if we should change anything within our config.

Reply:
Yup, that's subs watch app config, however that'll create nav url `insights/subscriptions/manifests` (in future `/rhel/subscriptions/manifests`) which should lead to this app, right?

Reply:
OK, yes that's correct; I had to go through docs to confirm this separate update was for us.  I just pushed an update for that.

Reply:
Nice! What do you want to have in the left navigation actually? `Manifests` or `Subscription Central`

Reply:
It should be Manifests.

Reply:
In that case you should remove the `title: Manifests` from Subs watch config and here change the `title` to `Manifests`.

Reply:
OK just updated that; thanks!